apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.ts

```diff
@@ -193,8 +228,17 @@ export class PaymentDialogComponent implements OnInit, OnDestroy {
      * @param {string} currencySymbol
      * @memberof PaymentDialogComponent
      */
-    private assignAmount(amount: number, currencySymbol: string): void {
-        this.paymentForm.get('amount')?.patchValue(amount);
+    private assignAmount(amount: number, currencySymbol: string, indexValue: number = 0): void {
```

Comment:
```suggestion
    private assignAmount(amount: number, currencySymbol: string, depositIndex: number = 0): void {
```
make changes in overall file

Reply:
Done